src/libxively/xi_globals.h

```diff
@@ -33,6 +33,9 @@ typedef struct
     char* str_account_id;
     char* str_device_unique_id;
     xi_backoff_status_t backoff_status;
+
+    char** updateable_files;
+    uint16_t updateable_files_count;
```

Comment:
shouldn't this be in the context?  What if two connections are built, and they both get file update available messages?  Wont' they conflict with each other?